config/webpack.prod.js

```diff
@@ -1,17 +1,20 @@
 const merge = require( 'webpack-merge' );
 const WebpackAssetsManifest = require( 'webpack-assets-manifest' );
-const path = require( 'path' );
 const common = require( './webpack.common.js' );
 const webpack = require( 'webpack' );
 const miniExtract = require( 'mini-css-extract-plugin' );
-const wpi18nExtractor = require( './bin/i18n-map-extractor.js' );
+const wpi18nExtractor = require( '../bin/i18n-map-extractor.js' );
```

Comment:
`bin` folder got renamed no?

Reply:
Not yet, I moved only `asset-dependency-maps.js` from `./bin` directory for now, because this file was related to webpack configs.
Didn't wanted to pollute PR's diff with too many changes, if you want, I can move the other files and references from `./bin` to `./config`

Reply:
ahh ok, what you have done is fine. 

Ideally we should move all config files into the new config folder you created, but that can happen in another PR.
I don't understand why that `bin` folder was created considering it does not contain any binary files !?!?

Reply:
Maybe `bin` from `trash bin/can` :thinking: